ejb3/src/main/java/org/jboss/as/ejb3/component/session/SessionBeanObjectViewConfigurator.java

```diff
@@ -115,7 +115,7 @@ public void configureDependency(final ServiceBuilder<?> serviceBuilder, final Co
 
                 if (componentMethod != null) {
                     if(!Modifier.isPublic(componentMethod.getModifiers())) {
-                        throw EjbLogger.ROOT_LOGGER.ejbBusinessMethodMustBePublic(componentMethod);
+                        EjbLogger.ROOT_LOGGER.ejbBusinessMethodMustBePublic(componentMethod);
                     }
```

Comment:
We may want to log it in debug level so users won't be alarmed by this message.

Reply:
Fixed